.buildkite/release-automation/forge_arm64.Dockerfile

```diff
@@ -11,14 +11,28 @@ set -euo pipefail
 
 apt-get update
 apt-get upgrade -y
-apt-get install -y curl zip
+apt-get install -y -qq \
+    curl zip clang-12
```

Comment:
could you move them back to one line? it does not seem that it needs wrapping yet.

and please remove the `-qq`, per `apt-get` man page:

````
you should never use -qq without a no-action modifier such as -d, --print-uris or -s as APT may decide to do something you did not expect
````

Reply:
fixed!